addons/maptools/XEH_postInitClient.sqf

```diff
@@ -32,7 +42,14 @@ GVAR(mapTool_moveToMouse) = true;  // used to display it in center of screen whe
     };
 }] call CBA_fnc_addPlayerEventHandler;
 
+addMissionEventHandler ["markerCreated", {
+    [FUNC(handlePlottingBoardMarkers), [_this, false]] call CBA_fnc_execNextFrame
+}];
+
+addMissionEventHandler ["markerDeleted", {
+    [FUNC(handlePlottingBoardMarkers), [[_this select 0, null, null, _this select 1], true] ] call CBA_fnc_execNextFrame
```

Comment:
```suggestion
    [FUNC(handlePlottingBoardMarkers), [[_this select 0, -1, objNull, _this select 1], true] ] call CBA_fnc_execNextFrame
```

`-1, objNull` could be replaced by `nil, nil`, as the function handles deleted markers at the top.